### DISCUSSION TITLE
Calling images returns NoneType

### DISCUSSION THREAD

**Discussion by ghost** [joalla/discogs_client#82] [Support] [answered]
Hi

I'm trying to use the api to gain the image URL but it's returning an object of NoneType. My code can be seen below. The artist call works fine, but images throws the NoneType error at me. If you were able to help me figure out where I've gone wrong, it would be appreciated.

Thanks!

```
paginatedResults = d.search(barcode=userBarcode).page(1)
   for i in paginatedResults:
      artist = i.artists[0].name
      cover = i.images[0]

**Comment by AnssiAhola** [comment, marked as answer]
Hey @DeputyBeagle 

Some releases do not have images attached to them. For example
https://www.discogs.com/release/22155439-Life-On-Planets-Stay
```python
> release = d.release("22155439")
> print(release.images)
> None
```

You should maybe just add a simple null/none check
```python
images = i.images
if images is not None:
    cover = images[0]
```

Hope that helps, Cheers!
Anssi
